src/dev/flang/be/interpreter/Interpreter.java

```diff
@@ -24,230 +24,100 @@
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.Map;
-
+import dev.flang.air.Clazz;                // NYI: remove this dependency
+import dev.flang.air.Clazzes;              // NYI: remove this dependency
+import dev.flang.ast.AbstractFeature;      // NYI: remove this dependency
+import dev.flang.ast.AbstractType;         // NYI: remove this dependency
+import dev.flang.ast.Types;                // NYI: remove this dependency
+import dev.flang.fuir.FUIR;
+import dev.flang.fuir.analysis.AbstractInterpreter;
 import dev.flang.util.ANY;
-import dev.flang.util.Errors;
-import dev.flang.util.FatalError;
 import dev.flang.util.FuzionOptions;
 import dev.flang.util.List;
 
-import dev.flang.fuir.FUIR;
-
-import dev.flang.air.Clazz;
-import dev.flang.air.Clazzes;
-import dev.flang.air.FeatureAndActuals;
-
-import dev.flang.ast.AbstractAssign; // NYI: remove dependency! Use dev.flang.fuir instead.
-import dev.flang.ast.AbstractBlock; // NYI: remove dependency! Use dev.flang.fuir instead.
-import dev.flang.ast.AbstractCall; // NYI: remove dependency! Use dev.flang.fuir instead.
-import dev.flang.ast.AbstractConstant; // NYI: remove dependency! Use dev.flang.fuir instead.
-import dev.flang.ast.AbstractCurrent; // NYI: remove dependency! Use dev.flang.fuir instead.
-import dev.flang.ast.AbstractFeature; // NYI: remove dependency! Use dev.flang.fuir instead.
-import dev.flang.ast.AbstractMatch; // NYI: remove dependency! Use dev.flang.fuir instead.
-import dev.flang.ast.AbstractType; // NYI: remove dependency! Use dev.flang.fuir instead.
-import dev.flang.ast.Box; // NYI: remove dependency! Use dev.flang.fuir instead.
-import dev.flang.ast.Check; // NYI: remove dependency! Use dev.flang.fuir instead.
-import dev.flang.ast.Env; // NYI: remove dependency! Use dev.flang.fuir instead.
-import dev.flang.ast.Expr; // NYI: remove dependency! Use dev.flang.fuir instead.
-import dev.flang.ast.If; // NYI: remove dependency! Use dev.flang.fuir instead.
-import dev.flang.ast.InlineArray; // NYI: remove dependency! Use dev.flang.fuir instead.
-import dev.flang.ast.Nop; // NYI: remove dependency! Use dev.flang.fuir instead.
-import dev.flang.ast.Tag; // NYI: remove dependency! Use dev.flang.fuir instead.
-import dev.flang.ast.Types; // NYI: remove dependency! Use dev.flang.fuir instead.
-import dev.flang.ast.Universe; // NYI: remove dependency! Use dev.flang.fuir instead.
-
-
-/**
- * Interpreter contains interpreter for Fuzion application that is present as
- * intermediate code.
- *
- * @author Fridtjof Siebert (siebert@tokiwa.software)
- */
 public class Interpreter extends ANY
 {
+  public static AbstractInterpreter<Value, Object> _ai;
+  // NYI: HACK:
+  public static Interpreter instance;
 
+  public final FUIR _fuir;
+  public final FuzionOptions _options_;
+  private final Excecutor _processor;
 
-  /*-----------------------------  statics  -----------------------------*/
```

Comment:
I think it helps to keep the general structure of classes, in particular of large classes. So please use these separators.